apps/condo/domains/common/utils/date.js

```diff
@@ -30,9 +30,16 @@ const getStartDates = (dateRaw) => {
     }
 }
 
+/**
+ * Returns current seconds count from Epoch start UTC
+ * @returns {number}
+ */
+const getCurrSec = () => Math.floor(Date.now() / 1000)
```

Comment:
It's implementation of timestamp. Maybe better name it something like `getTimestamp`?